common/testutils/utils.go

```diff
@@ -14,9 +15,53 @@ func ReadFile(filepath string) ([]byte, error) {
 	}
 	defer fileObj.Close()
 	// Read contents
-	byteValue, err := ioutil.ReadAll(fileObj)
+	byteValue, err := io.ReadAll(fileObj)
 	if err != nil {
 		return nil, err
 	}
 	return byteValue, nil
 }
+
+// testSetupEnvForGoogleCredentials creates a temporary file containing dummy service account JSON
+// then set the environment variable GOOGLE_APPLICATION_CREDENTIALS to point to the file.
+// This is useful for tests that assume Google Cloud Client libraries can automatically find
+// the service account credentials in any environment.
+// At the end of the test, the returned function can be called to perform cleanup.
+func TestSetupEnvForGoogleCredentials(t *testing.T) (reset func()) {
```

Comment:
Relocated this from `management-service/services/mlp_service_test.go` because this really helpful test helper is now used elsewhere besides the `services` package 😅